Library/Homebrew/global.rb

```diff
@@ -54,7 +54,13 @@ module Homebrew
 
 require "compat" unless ARGV.include?("--no-compat") || ENV["HOMEBREW_NO_COMPAT"]
 
-ORIGINAL_PATHS = ENV["PATH"].split(File::PATH_SEPARATOR).map { |p| Pathname.new(p).expand_path rescue nil }.compact.freeze
+ORIGINAL_PATHS = ENV["PATH"].split(File::PATH_SEPARATOR).map do |p|
+  begin
+                                                    Pathname.new(p).expand_path
+                                                  rescue
+                                                    nil
+                                                  end
+end.compact.freeze
```

Comment:
Wow, this particular change is atrocious looking.  What's with the gigantic indents?